android/java/res/layout/wallet_balance_layout.xml

```diff
@@ -165,35 +165,35 @@
         android:alpha="0.65"
         android:textSize="12sp"
         app:layout_constraintBottom_toBottomOf="@id/ads_monthly_statement_text"
-        app:layout_constraintStart_toEndOf="@id/ads_monthly_statement_text"
+        app:layout_constraintEnd_toStartOf="@id/bat_ads_balance_learn_more_text"
         app:layout_constraintTop_toTopOf="@id/ads_monthly_statement_text" />
 
     <TextView
         android:id="@+id/bat_ads_balance_learn_more_text"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
+        android:layout_width="62dp"
```

Comment:
we shouldn't set hard coded width here. it may cut the text for different translated text.

Reply:
ahh, It's my mistake.